libs/extensions/angular/image-banner/src/image-banner.component.scss

```
@@ -95,12 +95,15 @@ $container-query-breakpoint: 600px;
 
 .main-content {
   display: flex;
-  min-height: utils.line-height('l') * 3; // text content height should always be minimum 3 lines
   box-sizing: border-box;
   flex-direction: column;
   padding: utils.size('xs') 0 utils.size('xxs') utils.size('xxs');
   overflow: hidden;
 
+  &.has-minimum-height {
+    min-height: utils.line-height('l') * 3; // text content height should always be minimum 3 lines
+  }
```

Instead of using an input and class bindings to set a single style property, consider simply exposing this as a custom property. 

```suggestion
    min-height: var(--kirby-image-banner-min-height, #{utils.line-height('l') * 3}); // text content height should be minimum 3 lines by default
```